npc/re/jobs/novice/academy.txt

```diff
@@ -13,80 +13,226 @@
+	end;
+
+OnTouch_:
+	if (@cd + 15 > gettimetick(2))
+		end;
+	@cd = gettimetick(2);
+	npctalk "Urgh... please... help me...", "", bc_self;
+	sleep 7000;
+	npctalk "In order to talk with me.. click my body..";
+	end;
+}
+
+int_land,58,69,5	script	Sailor#int_land	4W_SAILOR,{
+	if (!checkquest(21002) || checkquest(21002) == 2) {
+		mes "[Sailor]";
+		mes "The ship is heading to Izlude soon~!";
+		mes "I'll let you know when we set sail~!";
+		close;
+	} else {
+		if (countitem(6008) >= 2) {
+			delitem 6008, countitem(6008);
+			completequest(21002);
+			mes "[Sailor]";
+			mes "Unbelievable, perfect! Any chance you want to join my crew?";
+			mes "Enough talking!!";
+			mes "Come on, we're ready to set sails!";
+			mes "Thank you so much!";
+			next;
+			mes "[Sailor]";
+			mes "If you want to sail with us to Izlude, jump on board!";
+			getexp 600,0;
+			specialeffect2 EF_HEAL2;
+			heal 9999, 0;
+			specialeffect EF_INCAGILITY, AREA, playerattached();
+			sc_start SC_INC_AGI, 240000, 10;
+			specialeffect EF_BLESSING, AREA, playerattached();
+			sc_start SC_BLESSING, 240000, 10;
+			next;
+			mes "[Sailor]";
+			mes "Oh, and take thos ^4d4dffMagnifier^000000.";
+			mes "When you hunt monsters you can find ^4d4dffunidentified items^000000.";
+			getitem 611, 5;
+			next;
+			mes "[Sailor]";
+			mes "Using a Magnifier will identify the item.";
+			next;
+			mes "[Sailor]";
+			mes "Maybe you have already some unidentified items from your fights with those porings?";
+			close;
+		}
+		mes "[Sailor]";
+		mes "These Porings stole my materials to repaid the ship. Do you help me to get them back?";
+		mes "There are plenty of Porings on this Island.";
+		next;
+		mes "[Sailor]";
+		mes "I really appreciate it if you help me.";
+		mes "Apparently I am not allowed to move or discover this island.";
+		mes "Thats why I ask you~";
+		close;
+	}
+
+OnInit:
+	questinfo 21002, QTYPE_QUEST, 1, Job_Novice;
+	end;
+}
+
+//== Captain Carocc (int_land) =================================
+int_land,78,103,5	script	Captain Carocc#int_land	4_M_REINDEER,{
+	if(checkquest(21002) == -1) {
+		mes "[Captain Carocc]";
+		mes "There are still people in the cabins?!";
+		mes "At least you are save.";
+		mes "Are you alright?";
+		next;
+		if (select("Im alright, but others need help.", "I think I am the last?") == 1) {
 			mes "[Captain Carocc]";
-			mes "Let me give you more details later.";
-			mes "See you back at the harbor deck.";
-			next;
+			mes "There are more people left?";
+			mes "I will send a rescue team to them";
+			mes "Thank you for your report";
+		} else {
 			mes "[Captain Carocc]";
-			mes "To get off this ship, you should enter the ^4d4fffShining Portal^000000 over there.";
-			mes "All transportation is made through the portals.";
-			setquest 7471;
-			completequest 7471;
-			close2;
-			cutin "", 255;
-			end;
+			mes "Alright, thats good.";
+			mes "The storm came unexpected but I never thought it hit us that hard.";
```

Comment:
`mes "The storm came unexpectedly but I never thought it would hit us that hard.";`